Problem: upgrade to prost 0.11 fails

https://github.com/crypto-com/tmkms-light/runs/7619988316?check_suite_focus=true#step:5:264
```
Compiling tmkms-light v0.3.0 (/home/runner/work/tmkms-light/tmkms-light)
[264](https://github.com/crypto-com/tmkms-light/runs/7619988316?check_suite_focus=true#step:5:265)
error[E0599]: no function or associated item named `decode_length_delimited` found for struct `tendermint_proto::privval::Message` in the current scope
[265](https://github.com/crypto-com/tmkms-light/runs/7619988316?check_suite_focus=true#step:5:266)
  --> src/rpc.rs:40:32
[266](https://github.com/crypto-com/tmkms-light/runs/7619988316?check_suite_focus=true#step:5:267)
   |
[267](https://github.com/crypto-com/tmkms-light/runs/7619988316?check_suite_focus=true#step:5:268)
40 |         let msg = PrivMessage::decode_length_delimited(msg.as_ref())
[268](https://github.com/crypto-com/tmkms-light/runs/7619988316?check_suite_focus=true#step:5:269)
   |                                ^^^^^^^^^^^^^^^^^^^^^^^ function or associated item not found in `tendermint_proto::privval::Message`
[269](https://github.com/crypto-com/tmkms-light/runs/7619988316?check_suite_focus=true#step:5:270)
   |
[270](https://github.com/crypto-com/tmkms-light/runs/7619988316?check_suite_focus=true#step:5:271)
   = help: items from traits can only be used if the trait is in scope
[271](https://github.com/crypto-com/tmkms-light/runs/7619988316?check_suite_focus=true#step:5:272)
help: the following trait is implemented but not in scope; perhaps add a `use` for it:
[272](https://github.com/crypto-com/tmkms-light/runs/7619988316?check_suite_focus=true#step:5:273)
   |
[273](https://github.com/crypto-com/tmkms-light/runs/7619988316?check_suite_focus=true#step:5:274)
5  | use prost::message::Message;
[274](https://github.com/crypto-com/tmkms-light/runs/7619988316?check_suite_focus=true#step:5:275)
   |
[275](https://github.com/crypto-com/tmkms-light/runs/7619988316?check_suite_focus=true#step:5:276)

[276](https://github.com/crypto-com/tmkms-light/runs/7619988316?check_suite_focus=true#step:5:277)
error[E0599]: no method named `encode_length_delimited` found for struct `tendermint_proto::privval::Message` in the current scope
[277](https://github.com/crypto-com/tmkms-light/runs/7619988316?check_suite_focus=true#step:5:278)
   --> src/rpc.rs:184:14
[278](https://github.com/crypto-com/tmkms-light/runs/7619988316?check_suite_focus=true#step:5:279)
    |
[279](https://github.com/crypto-com/tmkms-light/runs/7619988316?check_suite_focus=true#step:5:280)
184 |             .encode_length_delimited(&mut buf)
[280](https://github.com/crypto-com/tmkms-light/runs/7619988316?check_suite_focus=true#step:5:281)
    |              ^^^^^^^^^^^^^^^^^^^^^^^ method not found in `tendermint_proto::privval::Message`
[281](https://github.com/crypto-com/tmkms-light/runs/7619988316?check_suite_focus=true#step:5:282)
    |
[282](https://github.com/crypto-com/tmkms-light/runs/7619988316?check_suite_focus=true#step:5:283)
   ::: /home/runner/.cargo/registry/src/github.com-1ecc6299db9ec823/prost-0.10.4/src/message.rs:78:8
[283](https://github.com/crypto-com/tmkms-light/runs/7619988316?check_suite_focus=true#step:5:284)
    |
[284](https://github.com/crypto-com/tmkms-light/runs/7619988316?check_suite_focus=true#step:5:285)
78  |     fn encode_length_delimited<B>(&self, buf: &mut B) -> Result<(), EncodeError>
[285](https://github.com/crypto-com/tmkms-light/runs/7619988316?check_suite_focus=true#step:5:286)
    |        ----------------------- the method is available for `tendermint_proto::privval::Message` here
[286](https://github.com/crypto-com/tmkms-light/runs/7619988316?check_suite_focus=true#step:5:287)
    |
[287](https://github.com/crypto-com/tmkms-light/runs/7619988316?check_suite_focus=true#step:5:288)
    = help: items from traits can only be used if the trait is in scope
[288](https://github.com/crypto-com/tmkms-light/runs/7619988316?check_suite_focus=true#step:5:289)
help: the following trait is implemented but not in scope; perhaps add a `use` for it:
[289](https://github.com/crypto-com/tmkms-light/runs/7619988316?check_suite_focus=true#step:5:290)
    |
[290](https://github.com/crypto-com/tmkms-light/runs/7619988316?check_suite_focus=true#step:5:291)
5   | use prost::message::Message;
[291](https://github.com/crypto-com/tmkms-light/runs/7619988316?check_suite_focus=true#step:5:292)
    |
[292](https://github.com/crypto-com/tmkms-light/runs/7619988316?check_suite_focus=true#step:5:293)

[293](https://github.com/crypto-com/tmkms-light/runs/7619988316?check_suite_focus=true#step:5:294)
error: unused import: `prost::Message`
[294](https://github.com/crypto-com/tmkms-light/runs/7619988316?check_suite_focus=true#step:5:295)
 --> src/rpc.rs:6:5
[295](https://github.com/crypto-com/tmkms-light/runs/7619988316?check_suite_focus=true#step:5:296)
  |
[296](https://github.com/crypto-com/tmkms-light/runs/7619988316?check_suite_focus=true#step:5:297)
6 | use prost::Message as _;
[297](https://github.com/crypto-com/tmkms-light/runs/7619988316?check_suite_focus=true#step:5:298)
  |     ^^^^^^^^^^^^^^
[298](https://github.com/crypto-com/tmkms-light/runs/7619988316?check_suite_focus=true#step:5:299)
  |
[299](https://github.com/crypto-com/tmkms-light/runs/7619988316?check_suite_focus=true#step:5:300)
  = note: `-D unused-imports` implied by `-D warnings`
[300](https://github.com/crypto-com/tmkms-light/runs/7619988316?check_suite_focus=true#step:5:301)

[301](https://github.com/crypto-com/tmkms-light/runs/7619988316?check_suite_focus=true#step:5:302)
For more information about this error, try `rustc --explain E0599`.
[302](https://github.com/crypto-com/tmkms-light/runs/7619988316?check_suite_focus=true#step:5:303)
```

--- a/.github/workflows/tmkms.yml
+++ b/.github/workflows/tmkms.yml
@@ -57,6 +57,7 @@ jobs:
       # RUSTSEC-2020-0071: time crate used in upstream sgxs-* crates
       # RUSTSEC-2021-0127: serde_cbor used in upstream nsm-* crates
       # RUSTSEC-2020-0016,RUSTSEC-2021-0124: net2 used in mio from older tokio (used in sgx crates)
+      # RUSTSEC-2022-0041: old cross-beam used in sgx runner crate
       - run: >
           cargo audit --deny warnings
           --ignore RUSTSEC-2020-0071
@@ -65,6 +66,7 @@ jobs:
           --ignore RUSTSEC-2020-0016
           --ignore RUSTSEC-2021-0127
           --ignore RUSTSEC-2019-0036
+          --ignore RUSTSEC-2022-0041
   
   build:
     runs-on: ubuntu-latest

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -9,7 +9,7 @@ edition = "2021"
 [dependencies]
 ed25519-dalek = "1"
 flex-error = "0.4"
-prost = "0.10"
+prost = "0.11"
 serde = { version = "1", features = ["serde_derive"] }
 serde_json = "1"
 subtle-encoding = { version = "0.5", features = ["bech32-preview"] }

